.github/workflows/integration-test.yml

```diff
@@ -215,12 +215,12 @@ jobs:
       - name: Sharded Test (Ubuntu)
         if: runner.os != 'macOS' && runner.os != 'Windows'
         run: |
-          bash scripts/test-coverage-xml.sh integration ${{ inputs.test_environment }}
+          bash scripts/test-coverage-xml.sh integration ${{ inputs.test_environment }} 6 ${{ matrix.shard }}
```

Comment:
this starts to get a bit unclear to use. How would it look for a user running this... would `bash scripts/test-coverage-xml.sh` just work with some sane defaults for them?